var/spack/repos/builtin/packages/gccxml/package.py

```diff
@@ -36,3 +37,17 @@ class Gccxml(CMakePackage):
     patch('darwin-gcc.patch', when='%gcc platform=darwin')
     # taken from https://github.com/gccxml/gccxml/issues/11#issuecomment-140334118
     patch('gcc-5.patch', when='%gcc@5:')
+
+    depends_on('gcc@:4.9.999', type='run')
+
+    @run_after('install')
+    def post_install(self):
```

Comment:
rename function to indicate what it does. E.g. patch_runtime_compiler_path (if that is what it is doing)

Reply:
done